docs/typescript-sdks/apple/macos-node-api.md

```
@@ -0,0 +1,458 @@
+8. [Compatibility](#compatibility)
+9. [Known Limitations](#known-limitations)
+10. [Comparison with Alternatives](#comparison-with-alternatives)
+11. [Related Projects](#related-projects)
+
+---
+
+## Architecture
+
+### How It Works
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)